HarpGuru/Controls/ControlHelpers/index.ts

```diff
@@ -0,0 +1 @@
+export { getPropsForHarpStrata } from './getPropsForHarpStrata'
```

Comment:
Rename again to `reverseEngineerPropsForHarpStrata' or 'deducePropsForHarpStrata'